### ISSUE TITLE
Poppler

### ISSUE THREAD

**Issue by andered** [g-stavrakis/PDF_Text_Extraction#1]
This is a great tutorial - thanks. I am a novice, so I apologize if there is silly, but for the life of me I cannot get poppler to install properly (Windows 11, virtual python environment).  I have followed the relevant info on stack overflow (download the latest build, add the bin folder to Path) but it doesn't seem to work for me.  Perhaps this is my own computer weirdness, or something to do with the use of a virtual environment, but is there a trick to doing this on Windows 11? Or some other 'industry standard' approach that experts use? Thanks in advance for any guidance you can provide.

**Comment by g-stavrakis** [comment]
Hello, andered! I'm sorry for not getting back to you sooner. I just added a requirements.txt file in order to replicate the same environment if this helps. I am currently using a Mac and I created my environments through Anaconda. When I faced this issue I just ran the following command into my terminal to install Poppler and the problem resolved.

`conda install -c conda-forge poppler`

I found this command in pdf2image's ReadMe file here: https://github.com/Belval/pdf2image

I hope that this solves the problem. Thank you!
